MvvmCross/MvvmCross.csproj

```diff
@@ -18,6 +18,13 @@ This package contains the 'Core' libraries for MvvmCross</Description>
     <None Include="Platforms\**\*.cs" />
     <None Include="Resources\*.cs" />
     <Compile Remove="Resources\*.cs" />
+    <None Remove="Platforms\WinUi\Presenters\Attributes\MvxNewWindowPresentationAttribute.cs" />
```

Comment:
It shouldn't be neccessary to add anything in the csproj file. It is already configured to add whatever is in each Platforms folder

Reply:
Yep, was done automatically and I missed that. Removed it.